docs/source/beginner-tutorial.rst

```diff
@@ -101,14 +101,15 @@ interfaces: A command-line interface, ``IPython Notebook`` magic, and
 
 Ingesting Data
 ^^^^^^^^^^^^^^
-If you haven't ingested the data, run:
+If you haven't ingested the data, you'll need a `Quandl <https://www.quandl.com/>`__ API key to
```

Comment:
Same comment as before, we might want to link to https://docs.quandl.com/docs#section-authentication instead.